src/components/screens/EventRecords.js

```diff
@@ -257,6 +260,17 @@ export const EventRecords = ({ navigation, route }) => {
     showVolunteerEvents
   ]);
 
+  const filterTypes = useMemo(() => {
+    return filterTypesHelper({
+      data,
```

Comment:
<img width="373" alt="image" src="https://github.com/user-attachments/assets/d16631bc-347a-4c0d-8895-5ea0bc36b9d7">

i think this is because in filter types helper `usePermanentFilter` is called, so the `useMemo` here warns